build.yaml

```diff
@@ -2468,7 +2468,6 @@ steps:
               --durations=50 \
               --ignore=test/hailtop/ \
               --ignore=test/hail/matrixtable/test_file_formats.py \
-              -m qobtest \
```

Comment:
@daniel-goldstein did `qobtest` mean something different to `skip_unless_service_backend`?

Reply:
There was once a distinction that has become less and less relevant. Anything that's `skip_unless_service_backend` I think is something that is not supported in QoB.

`@qobtest` is a marker to specifically select the integration tests that we want to comprise the QoB integration test suite, even though other tests *can* on run QoB

Reply:
Ahh so if i give the `backend` marker with the `'batch'` backend this will do the same thing. Noice.